### ISSUE TITLE
Suggestion: actual pixel size scale.

### ISSUE THREAD

**Issue by MarioMey** [kgeogeo/VSE_Transform_Tools#9]
I know that this is not in development any more... but I post this here anyway.

It's great that the image, initially, is scaled to full render size. But sometimes, I want to work with the image as pixel scale: 1:1 (now I'm working with 8bits games images).

Alt-S resets scale to full render size. Maybe Alt-Shift-S would scale to 1:1.

Thanks for this addon, anyway.

**Comment by TheDuckCow** [comment]
I agree, it's a useful thing - useful enough that I created my own mini addon to do this, but honestly would love it if it were in life's here - and the shortcut key's not a bad idea either

https://github.com/TheDuckCow/Blender-Render-Match-VSE-Strips
(Note this actually changes resolution, not the actual image/strip scale. But similar need)

**Comment by MarioMey** [comment]
Maybe I didn't explain myself well... 

For example, I work with 384x224 render size (Street Fighter II arcade version real resolution). I import a Guile sprite, ripped off from original (just image that). Sprite PNG is 64x100 px. When I import to VSE, it appear full render size: X and Y: huge and with bad proportion. When I apply VSE Transform Tool to this strip, its proportion is fixed... but it is at full render size.

I would like to make this "sprite" with 64x100 size (as original)... But output render size to be 384x224 (not to touch this config).

**Comment by parameciostudio** [comment]
If you want to mantain the original size of the image, you can open the 
Properties Panel and check the option Strip Input > Image Offset before 
applying the Transform strip.

NOTE: This technique works only if the image is smaller than the render size. Bigger images will be cropped.
I've tried different combinations, using both Image Offset and Image Crop checkboxes, but the Transform Strip takes into account only the pixels visible on the screen.

**Comment by MarioMey** [comment]
Yes, you are right. Before reading your "NOTE", I tried to put an image larger than render size. Y checked Image Offset and then applied Tranform tool. The image changed its Y scale to smaller than original. The image **is not cropped** but, as I just told, the original proportion is not respected.

Also, if the image is smaller that render size, when applying Transform Tool, image position is modified from bottom-left to a bit higher. Maybe a minor bug.

I think this "Original image size" should be an additional option. And then, Alt-S should scale the image to original pixels. Now, even I check Image Offset, Alt-S make the image at render size.

**Comment by NathanLovato** [comment]
Hey, I'm working on a [Blender VSE add-on](https://github.com/GDquest/Blender-power-sequencer) that should do what you need. It comes with a `PS.Add transform effect` operator that's here to replace that of VSE_Transform_Tools. Although the add-on is fairly stable, you'll see it lacks docs. But I edit all my [tutorials](http://youtube.com/c/gdquest) with it, so it's only getting better.

And I'm looking to slowly merge functionality from this add-on and hopefully improve them along the way, as its development seems to have stopped.

**Comment by MarioMey** [comment]
I downloaded and testing it... but no, it doesn't do what I need.

I removed Transform Tool addon and installed yours. I add an image, press Space, select PS.Add transform effect and the image is full render size. Even if I check Image Offset first (I noted that, by adding the transform, this option is checked automatically).

Then... I don't know what to do. GRS keys don't work (should them?).

Also, I suggest you to check the other issue that I reported in this github, about using original pixels.

Thanks for your reply, anyway!

PS: not any docs? not even "add transform effect and press some key to do some thing"?

**Comment by NathanLovato** [comment]
Okay I'd have to check it again then. The add-on doesn't replace VSE transform tools atm, it has different features. And barely any docs at all - again, it's something I did for myself initially, and I already have too much work to refine it for public use now. It'll come out eventually though.

**Comment by NathanLovato** [comment]
If you can open an issue on the Power Sequencer repo, I'll look into it this week.

**Comment by MarioMey** [comment]
I don't have doubts that Power Sequencer is a great tool... but, for now, I don't know how to try it... and don't have the time to do a reverse engineering!

Also, I can't open an issue in PS... because I don't know what it really does... and then, what it doesn't do.

I just clicked in "Watch" in PS GitHub :).

**Comment by doakey3** [comment]
I have started developing [this version of the addon](https://github.com/doakey3/vse_transform_tools). My crop tool allows you to crop the image in place and pressing Alt S will return the image to it's original scale. Is that what you're looking for @MarioMey ?

**Comment by MarioMey** [comment]
Hey, good job, doakey3! It would be great if you list the fixes and new features in comparison with the original non-developed-any-more version.

**Comment by MarioMey** [comment]
Sorry, I didn't download your fork. I don't have time for now...

**Comment by doakey3** [comment]
check the [youtube video](https://www.youtube.com/watch?v=iDqIKDiUYrg). Also the [change log](https://github.com/doakey3/VSE_Transform_Tools/releases)
